__tests__/app.test.js

```diff
@@ -146,4 +146,14 @@ describe("/api/app", () => {
         expect(body.msg).toBe("No reviews found for review_id: 10000");
       });
   });
+  test("Post a new entry into the comment", () => {
+    return request(app)
+      .post("/api/reviews/5/comments")
+      .send({ author: "mallionaire", body: "hi i'm mallionaire" })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toHaveProperty("author");
```

Comment:
We also expect a comment from the database to have extra properties, like created_at and comment_id; make sure you're checking against all of them here. Also, we know that the username and body should be specific values ('mallionaire' and 'hi i'm mallionaire') so let's test for these specifically